templates/main-org.template.yaml

```diff
@@ -880,6 +883,9 @@ Resources:
         SecurityToolingAccountRegion: !Ref Section1ToolingAndLoggingRegion
         LogArchivingAccountRegion: !Ref Section1ToolingAndLoggingRegion
         OrganizationRootID: !Ref Section1OrganizationRootID
+        CreateConfigRecorderRole: !If [ enable_aws_config, 'Yes','No' ]
+        ConfigRegions: !Sub '${Section4SecurityHubRegionsToEnable},${Section6FirewallManagerPolicyRegions}'
```

Comment:
what if both Section4SecurityHubRegionsToEnable and Section6FirewallManagerPolicyRegions contains some common region ? Will the code work even with duplicate comma separated region values ?

Reply:
it's no problem. I have merged the duplicated regions